index.d.ts

```diff
@@ -0,0 +1,80 @@
+import { EventEmitter } from "events";
+import { User, Guild, Channel, Client, ClientOptions } from "eris";
+
+declare namespace ErisSharder {
+  interface Sharder extends EventEmitter {
+    shardCount: number;
+    firstShardID: number;
+    lastShardID: number;
+    clusterCount: number;
+    clusterTimeout: number;
+    token: string | false;
+    clusters: Map<>;
+    workers: Map<>;
+    queue: ErisSharderQueue;
```

Comment:
Not sure what type this is

Reply:
This is a placeholder for typings that I need to make for the Queue class in the utils folder.